src/XRP/model/xrp-memo.ts

```diff
@@ -20,6 +26,48 @@ export default class XRPMemo {
     return new XRPMemo(data, format, type)
   }
 
+  /**
+   * Constructs an XRPMemo from raw values that represent the fields of a memo.
+   *
+   * @param data an optional Uint8Array for the data in the memo.
+   * @param format an optional Uint8Array for the format in the memo.
+   * @param type an optional Uint8Array for the type in the memo.
+   * @returns an XRPMemo with its fields set via the arguments matching the names of said fields.
+   * @see https://github.com/ripple/rippled/blob/develop/src/ripple/proto/org/xrpl/rpc/v1/transaction.proto#L80
+   */
+  public static fromRaw(
```

Comment:
I think you can just use the `constructor` on line 79? No reason that we can't make that public.

Reply:
I just figured if we were already following a factory pattern for construction it was worth keeping. Given the relative incompatible yet similar arguments to construct an instance, my gut was that a factory method was more descriptive about the intention than a raw constructor.

Reply:
I went back and removed this actually.